#! /bin/sh -e
## 02_cdrecord_default_conf.dpatch by Zack Cerza <zcerza@coe.neu.edu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Set up reasonable default values in the cdrecord config

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad cdrtools-2.01.01~/cdrecord/cdrecord.dfl cdrtools-2.01.01/cdrecord/cdrecord.dfl
--- cdrtools-2.01.01~/cdrecord/cdrecord.dfl	2006-03-19 21:08:36.000000000 +0100
+++ cdrtools-2.01.01/cdrecord/cdrecord.dfl	2006-03-19 21:10:46.260143728 +0100
@@ -7,7 +7,7 @@
 # The default device, if not specified elswhere
 #
 #CDR_DEVICE=yamaha
-CDR_DEVICE=default
+CDR_DEVICE=cdrom
 
 # 
 # The default speed, if not specified elswhere
@@ -39,12 +39,13 @@
 #
 # drive name	device	speed	fifosize driveropts
 #
-default=	USCSI:1,0,0	-1	-1	burnfree
-ricoh-u=	USCSI:0,2,0	-1	-1	burnfree
-teac=		1,3,0	-1	-1	""
-panasonic=	1,4,0	-1	-1	""
-plextor=	1,4,0	-1	-1	""
-sanyo=		1,4,0	-1	-1	burnfree
-yamaha=		1,5,0	-1	-1	""
-cdrom=		0,6,0	2	1m	""
-remote=		REMOTE:rscsi@somehost:1,0,0	16	16m	burnfree
+#default=	USCSI:1,0,0	-1	-1	burnfree
+#ricoh-u=	USCSI:0,2,0	-1	-1	burnfree
+#teac=		1,3,0	-1	-1	""
+#panasonic=	1,4,0	-1	-1	""
+#plextor=	1,4,0	-1	-1	""
+#sanyo=		1,4,0	-1	-1	burnfree
+#yamaha=		1,5,0	-1	-1	""
+#cdrom=		0,6,0	2	1m	""
+#remote=		REMOTE:rscsi@somehost:1,0,0	16	16m	burnfree
+cdrom=          /dev/cdrw      -1      -1      burnfree
